pages/directory/[orgName].tsx

```diff
@@ -60,3 +51,36 @@ const OrganizationPage = () => {
 };
 
 export default OrganizationPage;
+
+export const getStaticPaths: GetStaticPaths = async () => {
+  const db = new DbProvider();
+  const clubs = await db.getAllClubs();
```

Comment:
Is it correct that this requests a list of all orgs when loading any individual page? It seems like it'll be important to ensure we're caching this endpoint when possible for performance and reducing API calls when reasonable.

Reply:
This function only runs at build time, it ensures that a page exists for all clubs in the database. Caching should not be necessary

Reply:
Thanks for clarifying that! I had a misconception about this, and have re-read the docs to make sure I understood this correctly.